test/TestNamespaceTracker.hs

```diff
@@ -308,16 +320,30 @@ testClientSetForwarded = do
     assertOnlyKeysInMap [alice] response
     assertMapValue alice [FRBOwner $ OwnerRequestBundle [] [dum helloP Set]] response
 
-trackerHelper = trackerHelper' mempty mempty
+pubTrackerHelper ::
+    forall i.  (Ord i, Show i) =>
```

Comment:
We don't need the `forall` here. `trackerHelper'` needed it because it used the `i` it bound in the function signature in another signature in a `where` clause.